### PR TITLE
Dockerbuild

### PR DESCRIPTION
**Pull request by joshuacox**
I've added a method for building with docker, it's a bit more lightweight than a VM and everything is discarded neatly aferwards.

### CODE REVIEW

**Comment by joshuacox** [comment]
I just did a build on alpine linux to test there:

```
2023-11-01 10:47:17,660: Results are in /live/tmp/lmc-result-6me1vx2f
+ sudo find /live/tmp -name vojtux_38.iso -exec mv -t /output '{}' +
real    39m 20.97s
user    0m 0.13s
sys     0m 0.10s
+ pwd
+ du -sh /root/vojtux/output
2.3G    /root/vojtux/output
+ du -sh ./tmp
8.0K    ./tmp
+ pwd
+ sudo find /root/vojtux/output -name vojtux_38.iso
/root/vojtux/output/vojtux_38.iso
+ rm -rf ./tmp
```

in addition, on my personal laptop it builds in about ten minutes, I think that the tmp directory gets to be around 15 GB during the build so you need the appropriate amount of space there.  For example, I saw failures on a machine where this was tmpfs and half of ram (the tmpfs default) was not enough to accommodate the build.

**Comment by joshuacox** [comment]
I think that eventually we could get a github action to automatically build this and host the binary.

File: build.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# but does not present to be a problem till after the squashfs runtime
+time docker run \
+	--privileged=true \
+	-it --rm \
+	-v /dev:/dev \
+	-v ./tmp:/live/tmp \
+	-v ./output:/output \
+	vojtux:latest
+
+du -sh $(pwd)/output
```
Comment (vojtapolasek) [inline build.sh:26]: I think this needs sudo... or maybe it can be removed.

**Review by vojtapolasek — CHANGES_REQUESTED** [review]
Hello @joshuacox  and thank you so much for this PR. This greatly simplifies testing of new images. Could you please just move all the stuff into some directory so that the root of the repo doesn't become cluttered?
Thank you.

File: build.sh (hunk 1 above)
Comment (joshuacox) [reply build.sh]: yep, that was just my curiosity, I will just discard a lot of the noise.

**Comment by joshuacox** [comment]
@vojtapolasek you bet, let me know what I can do to improve the 'readability' of my code. I realize that some of my habits on formatting my code may just result in noise to you and your screenreader.

**Review by vojtapolasek — APPROVED** [review]

**Comment by vojtapolasek** [comment]
Hello @joshuacox  and thank you for changes. NO worries, your code is well readable with a screenreader. Thank you for this improvement.

**Pull request merged**